Formula/rat.rb

```diff
@@ -0,0 +1,27 @@
+class Rat < Formula
+  desc "Compose shell commands to build terminal applications"
+  homepage "https://github.com/ericfreese/rat/archive/v0.0.2.tar.gz"
```

Comment:
It broke earlier because you put the wrong thing in the wrong place. You want to do:

```ruby
homepage "https://github.com/ericfreese/rat"
url "https://github.com/ericfreese/rat/archive/v0.0.2.tar.gz"
sha256 "9da8d67a1e16056e3f05cdf536766e5571ede54a6c241c941d6c74eee5d64d31"
```
and so on.

Reply:
Hmm.. from what I can tell, that's exactly what I did: https://github.com/Homebrew/homebrew-core/pull/19383#discussion_r144444392